gremlin-groovy/src/test/java/org/apache/tinkerpop/gremlin/groovy/engine/ScriptEnginesTest.java

```diff
@@ -194,8 +195,8 @@ public void shouldFailUntilImportExecutes() throws Exception {
         threadEvalAndTriggerImport.join();
         threadImport.join();
 
-        assertTrue(successes.intValue() > 0);
-        assertTrue(failures.intValue() >= 500);
+        assertTrue("Success: " + successes.intValue() + " - Failures: " + failures.intValue(), successes.intValue() > 0);
```

Comment:
I tend to convert `assertTrue()` to hamcrest based asserts like `assertThat(success.intValue(), isGreaterThan(0))` which yields a better output on failure which i assume is what you are after here.